ci_diff_helper/environment_vars.py

```diff
@@ -145,3 +145,45 @@
 We only expect this environment variable to be set during a
 build that is a part of a pull request from a fork.
 """
+
+GITHUB_PULL_REQUEST_NUMBER = 'GITHUB_PULL_REQUEST_NUMBER'
+"""The ID of the PR that started the current build.
+
+We only expect this environment variable to be set during a
```

Comment:
I would back off on "only" to keep the doc from getting stale / avoid overstating if other services support it but we don't use it.